samples/exemplar/build.gradle.kts

```diff
@@ -1,20 +1,26 @@
 plugins {
   kotlin("jvm")
+  application
   id("com.squareup.wire")
 }
 
+val applicationMainClass = "com.squareup.exemplar.ExemplarServiceKt"
```

Comment:
the Application plugin makes it easier to run the sample services in IntelliJ or with Gradle